pkg/profile/artifact_test.go

```diff
@@ -423,7 +451,7 @@ var _ = Describe("Profile", func() {
 				Expect(profilesv1.AddToScheme(scheme)).To(Succeed())
 				p = profile.New(pDef, pSub, fakeClient, logr.Discard())
 				err := p.CreateArtifacts(ctx)
-				Expect(err).To(MatchError(ContainSubstring("failed to create GitRepository resource")))
+				Expect(err).To(MatchError(ContainSubstring("failed to create resource: no kind is registered for the type v1beta1.GitRepository in scheme")))
```

Comment:
@aclevername Curious. This suddenly is throwing a different error. I'll investigate why but it's not obvious at first glance. I changed too much. :D

Reply:
Oh I think I know actually.

Reply:
Yeah, I think this is sort of fine? Now the log is more precise and tells you that the kind doesn't exist. I can add the Kind that it was trying to create into the Error as well. That helps too.

Reply:
Now it will say:
```
failed to create v1beta1.GitRepository
```
instead of
```
failed to create resource
```

@aclevername Is that okay?

Reply:
I think its worth mentioning which resource error'd? Helps for debugging

Reply:
Yep added that. Now this looks like this:

```
failed to create HelmRelease: no kind is registered for the type v2beta1.HelmRelease
```